rotation.txt

```diff
@@ -1,38 +1,55 @@
-Deepwind Jungle
-Sandstone 2
-WarlockDTW
-Ricco
-Rooted Frontier
-Ring Race
-Twin Peaks
-Limbo II
-Rooted
-Ascension
-Senex
-Brisked KOTH
-Desert Country
-Stonehenge
-Turf Wars
-Race for Victory 2
+Destiny
+Pluto
```

Comment:
This map takes quite a while, especially if there's a bunch of players online. Often, Blitz maps cause the player count to go down as players can't join midgame, and especially with a map with 3 lives it would probably only cost us a bunch of players if we had this map in the rotation.